### ISSUE TITLE
Stacked Area Chart - missing 'stroke' for each dataset

### ISSUE THREAD

**Issue by deandwill** [carbon-design-system/carbon-charts#671]
#### I am submitting a...

- [ ] Feature request
- [ ] Design defect
- [x] Source code defect
- [ ] Demo/documentation defect
- [ ] Other

### Issue description

The 'stroke' outline for each dataset is missing 

#### Steps to produce the issue

If you start with https://carbon-design-system.github.io/carbon-charts/?path=/story/area--stacked-area-time-series … and you change “date” to “key” ( eg: https://codesandbox.io/s/cranky-payne-9yxq3?file=/src/index.js )  the ‘stroke’ outline for each dataset disappears 

#### Current behavior
<img width="552" alt="Screen Shot 2020-06-17 at 10 47 43 AM" src="https://user-images.githubusercontent.com/446838/84916979-c1b50f00-b08c-11ea-949f-bf7defcabf2c.png">

#### Expected behavior
<img width="556" alt="Screen Shot 2020-06-17 at 10 47 24 AM" src="https://user-images.githubusercontent.com/446838/84917088-e01b0a80-b08c-11ea-89df-b014d907402b.png">

**Comment by theiliad** [comment]
@natashadecoste could you please look into this issue?

**Comment by Yohanna** [comment]
I found a similar issue that I think is related to this bug as well. In the [SimpleBarChart story](https://carbon-design-system.github.io/carbon-charts/?path=/story/bar-horizontal--simple-horizontal-bar-discrete), if you replace the `data` with this:

```jsonc
[
  {
    "key": "Qty", // Notice the "key"
    "value": 65000
  }
]
```

It crashes the chart.

Let me know if I should open a separate issue for this.

**Comment by natashadecoste** [comment]
this should have been fixed with #678
